content/dstu2/contract.md

```diff
@@ -42,11 +42,11 @@ Notes:
 ### Response
 
 Search for all Contracts subject to a specific patient
-<%= headers 200, GET: '[...]/Contract?subject:Patient=14067775' %>
+<%= headers 200, GET: '[...]/Contract?subject:Patient=4598008' %>
```

Comment:
The changed ids here still aren't in sync with the ids currently in the contract JSON examples